config/default.js

```diff
@@ -61,5 +61,6 @@ module.exports = {
     domain: 'mozilla-iot.org',
     pagekite_cmd: './pagekite.py',
     port: 443
-  }
+  },
+  bcryptRounds: 10
```

Comment:
I think it would be better to allow this to not be specified and just use the default. I know that generally the default changes every couple years so that bcrypt takes approx the same amount of time with new compute hardware. By explicitly setting this, we'll eventually fall behind.